playbooks/dbgsym-kernel.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: add debug symbol archive key
+  apt_key: keyserver=keyserver.ubuntu.com id=2512191FEF8729D6E5AF414DECDCAD72428D7C01
```

Comment:
As far as I know this is the only 'right' way to get an Ubuntu signing key, and since it didn't fit the key URL/repo URL pattern I didn't put it in the apt_repos role.